assets/source/setup-guide/app/style.scss

```diff
@@ -590,6 +590,18 @@ $root: ".woocommerce-setup-guide";
 			display: none;
 		}
 
+		.components-card__footer {
+			flex-direction: row;
+
+			.link {
+				text-decoration: none;
+
+				&:hover {
+					text-decoration: underline;
+				}
+			}
```

Comment:
💅 

I see that the rest of the links are actually underlined. Why we are removing the underline on this one is it a requirement?

Given that, In case it is a requirement:  

Why we are using this custom  `.link` selector instead of `.components-external-link` component's default?

Reply:
Ha, I wasn't aware there is a default class used by `<ExternalLink>` component. 😅 TIL, thanks @puntope!

I will fix it soon and request your review again. 😄 

A side question: any tips to find out if there's a default class for a component? I've found two ways but would love to learn more.

- Open chrome devtool and inspect the target element, there's a class in the `<a>` tag, e.g.
```html
<a class="components-external-link link" href="https://ads.pinterest.com/" target="_blank" rel="external noreferrer noopener">
```
- In https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/external-link/index.js, it clearly uses `components-external-link` classname there.

Reply:
Forgot to answered this:
> I see that the rest of the links are actually underlined. Why we are removing the underline on this one is it a requirement?

It's based on the design screenshot in #252, from the screenshot the link is not underlined so I assume it's a requirement.

Reply:
> A side question: any tips to find out if there's a default class for a component? I've found two ways but would love to learn more.

I think those options are good. Instead of going to Github. What I normally do is click on the import element in the IDE and then it shows me the src code of that import. 

<img width="1037" alt="Screenshot 2021-11-24 at 10 51 32" src="https://user-images.githubusercontent.com/5908855/143215285-b01225b2-caaa-4977-9b1a-3aace613a3a4.png">

Reply:
Fixed in 8c68886.